### DISCUSSION TITLE
Failing to plot UMAP after integration

### DISCUSSION THREAD

**Discussion by VDD58** [GreenleafLab/ArchR#1498] [Questions / Documentation]
Hi, 
I have 10x scATAC data that i am analyzing using ArchR. 
Currently I am trying to perform an unconstrained integration of scRNA data (SMART-seq) but unfortunately I am unable to plot an UMAP after integration. I receive the following error:

Error in getEmbedding(ArchRProj, embedding = embedding, returnDF = TRUE): Embedding not in computed embeddings, Current ones are : UMAPHarmony_res_0.3,UMAP_res_0.3
Traceback:

1. plotEmbedding(proj2.1_integrated, colorBy = "cellColData", name = "predictedGroup_TTT", 
 .     pal = pal)
2. getEmbedding(ArchRProj, embedding = embedding, returnDF = TRUE)
3. stop(paste0("Embedding not in computed embeddings, Current ones are : ", 
 .     paste0(names(ArchRProj@embeddings), collapse = ",")))

I really do not understand why this is happening considering that both embeddings are present in the ArchR project.
proj2.1_integrated@embeddings
List of length 2
names(2): UMAPHarmony_res_0.3 UMAP_res_0.3

If you have any suggestions, they will be highly appreciated. 
Valeriya

**Comment by rcorces** [comment]
The default value for the `embedding` parameter is "UMAP". From what I can tell, you havent provided a value for that parameter so it has defaulted to the name "UMAP" which isnt available in your project.

**Reply by VDD58** [reply]
Thank you! 
I have included embedding = "UMAP_res_0.3" and of course it worked! 
Thank you again!
